spec/europeana/blacklight/document/more_like_this_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+RSpec.describe Europeana::Blacklight::Document::MoreLikeThis do
+  subject { Europeana::Blacklight::Document.new(source) }
+  let(:source) { {} }
+
+  describe '#more_like_this' do
+    context 'without blacklight_config' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.